pkg/types/gcp/validation/machinepool_test.go

```diff
@@ -48,7 +48,7 @@ func TestValidateMachinePool(t *testing.T) {
 					DiskType: "pd-",
 				},
 			},
-			expected: `^test-path\.diskType: Unsupported value: "pd-": supported values: "pd-ssd", "pd-standard"$`,
+			expected: `^test-path\.diskType: Unsupported value: "pd-": supported values: "pd-ssd", "pd-balanced", "pd-standard"$`,
```

Comment:
Ordering is based on alphabetical so this needs to be.
```suggestion
			expected: `^test-path\.diskType: Unsupported value: "pd-": supported values: "pd-balanced", "pd-ssd", "pd-standard"$`,
```